app/views/userTournament/list.scala

```diff
@@ -18,17 +18,17 @@ object list:
       pager: Paginator[lila.tournament.LeaderboardApi.TourEntry],
       count: String
   )(using Lang) =
-    if pager.nbResults == 0 then div(cls := "box-pad")(u.username, " hasn't played in any tournament yet!")
+    if pager.nbResults == 0 then div(cls := "box-pad")(trans.arena.xHasNotPlayedInAnyTourYet(u.username))
     else
       div(cls := "tournament-list")(
         table(cls := "slist")(
           thead(
             tr(
               th(cls := "count")(count),
-              th(h1(userLink(u, withOnline = true), " tournaments")),
-              th("Games"),
-              th("Points"),
-              th("Rank")
+              th(h1(trans.arena.xTournaments.txt(userLink(u, withOnline = true)))),
```

Comment:
this is also broken. Please try your change when publishing a pull request, or set it to "DRAFT" until it's tested.